discovery/src/test/java/com/ibm/watson/discovery/v1/DiscoveryServiceIT.java

```diff
@@ -146,7 +146,7 @@ public static void cleanupClass() throws Exception {
   public void setup() throws Exception {
     super.setUp();
     String apiKey = getProperty("discovery.apikey");
-    String url = getProperty("discovery.url");
+    String url = "https://api.us-south.discovery.watson.cloud.ibm.com";
```

Comment:
Should this be hardcoded? Eventually we should update the credentials to use the proper urls.

Reply:
updated! [b66e5d4](https://github.com/watson-developer-cloud/java-sdk/pull/1130/commits/b66e5d40c76cf6fdd9e53c4e08060d4b70dfc3cf)